packages/swapper/src/swappers/zrx/ZrxBuildQuoteTx/ZrxBuildQuoteTx.ts

```diff
@@ -131,8 +129,8 @@ export async function ZrxBuildQuoteTx(
       rate: data.price,
       depositAddress: data.to,
       feeData: {
-        fee: new BigNumber(estimatedGas || 0)
-          .multipliedBy(new BigNumber(data.gasPrice || 0))
+        fee: bnOrZero(estimatedGas || 0)
+          .multipliedBy(bnOrZero(data.gasPrice || 0))
```

Comment:
Same nit: Nit: remove `|| 0`

It likely doesn't matter because we're multiplying here, as opposed to dividing, but I think we want to make sure these remain integers, so doing a `decimalPlaces(0)` or `dp(0)` might prevent some rare bugs.